modules/simple_forms_api/app/models/simple_forms_api/vba_21_10210.rb

```diff
@@ -37,7 +38,11 @@ def submission_date_config
       }
     end
 
-    def track_user_identity; end
+    def track_user_identity
+      identity = "#{data['claimant_type']} #{data['claim_ownership']}"
+      StatsD.increment("#{STATS_KEY}.#{identity}")
+      Rails.logger.info('Simple forms api - 21-10210 submission user identity', identity:)
+    end
```

Comment:
I would love to hear your thoughts on how we can make this a shared callback in the future. I see that so far this is the second time this method is actually being populated but it's doing basically the same thing with some variance. It'd be nice to do some tweaking and ensure it's always called given our focus on logs.

That conversation is for a different time, though.

Reply:
Yep! Would be happy to consider other ways of doing this. The pattern I have here is just what happened to pop into my mind when I had to implement it the first time. 😄